quantities/quantities/Area.cs

```diff
@@ -8,18 +8,18 @@ namespace Quantities.Quantities;
 
 public readonly struct Area : IQuantity<Area>, IArea
     , IFactory<Area>
-    , IFactory<ISquareFactory<Area, IArea, ILength>, SquareFactory<Area, PowerFactory<Area, SquareTo, ILength>, IArea, ILength>, SquareFactory<Area, PowerFactory<Area, SquareCreate, ILength>, IArea, ILength>>
+    , IFactory<ISquareFactory<Area, IArea, ILength>, SecondPower<To, Area, IArea, ILength>, SecondPower<Create, Area, IArea, ILength>>
```

Comment:
Maybe rename `ISquareFactory` to `ISecondPower`?

Reply:
Let's resolve this in the discussion about the cubic factory.